src/impl/track.cpp

```diff
@@ -182,10 +182,11 @@ bool Track::transportSend([[maybe_unused]] message_ptr message) {
 void Track::setMediaHandler(shared_ptr<MediaHandler> handler) {
 	{
 		std::unique_lock lock(mMutex);
```

Comment:
Should this function `mRtcpHandeler->onOutgoing(nullptr)` before setting the new callback?

Reply:
Yes, I think it could be good to ensure they is only one handler outputting something.

Reply:
Added in https://github.com/paullouisageneau/libdatachannel/pull/369.